.github/workflows/upgrade-python-requirements.yml

```diff
@@ -0,0 +1,71 @@
+name: Upgrade Requirements
+
+on:
+  push:
+    branches:
+      - '**'
+  schedule:
+    # will start the job at 3:15 UTC every Wednesday
+    - cron: "15 3 * * 3"
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: "Target branch to create requirements PR against"
+        required: true
+        default: 'master'
+
+jobs:
+  upgrade_requirements:
+    runs-on: ubuntu-20.04
+
+    strategy:
+      matrix:
+        python-version: ["3.8"]
+
+    steps:
+      - name: setup target branch
+        run: echo "target_branch=$(if ['${{ github.event.inputs.branch }}' = '']; then echo 'master'; else echo '${{ github.event.inputs.branch }}'; fi)" >> $GITHUB_ENV
+
+      - uses: actions/checkout@v1
+        with:
+          ref: ${{ env.target_branch }}
+      
+      - name: setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: make upgrade
+        run: |
+          cd $GITHUB_WORKSPACE
+          make upgrade
+
+      - name: setup testeng-ci
+        run: |
+          git clone https://github.com/edx/testeng-ci.git
+          cd $GITHUB_WORKSPACE/testeng-ci
+          pip install -r requirements/base.txt
+      - name: create pull request
+        env:
+          GITHUB_TOKEN: ${{ secrets.REQUIREMENTS_BOT_GITHUB_TOKEN }}
+          GITHUB_USER_EMAIL: ${{ secrets.REQUIREMENTS_BOT_GITHUB_EMAIL }}
+        run: |  
+          cd $GITHUB_WORKSPACE/testeng-ci
+          python -m jenkins.pull_request_creator --repo-root=$GITHUB_WORKSPACE \
+          --target-branch="${{ env.target_branch }}" --base-branch-name="upgrade-python-requirements" \
+          --commit-message="chore: Updating Python Requirements" --pr-title="Python Requirements Update" \
+          --pr-body="Python requirements update.Please review the [changelogs](https://openedx.atlassian.net/wiki/spaces/TE/pages/1001521320/Python+Package+Changelogs) for the upgraded packages." \
+          --user-reviewers="" --team-reviewers="community-engineering" --delete-old-pull-requests
+
+      - name: Send failure notification
```

Comment:
We have removed this on our repos, relying on CODEOWNERS. Please see https://github.com/edx/xblock-lti-consumer/blob/master/.github/workflows/upgrade-python-requirements.yml

Reply:
@sarina I don't think this is the same thing.
- `CODEOWNERS` tags folks when PRs are _successfully_ created
- This build step (that's since been removed) notifies folks when this upgrade job _fails_ to create a PR in the first place.

I don't think we want to remove this notification. If the jenkins job were to start failing, we wouldn't know; we'd just stop getting upgrade PRs.
Given that there could be some weeks w/out upgrades, I wouldn't want us to rely on human memory and pattern recognition to remember/notice that it's been too long since we've gotten upgrades on a particular repo.